qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -190,6 +190,33 @@ def current_url(self):
         qtutils.ensure_valid(url)
         return url
 
+    def check_forms_can_be_destroyed(self, tab):
+        """If needed ask user for confirmation to close a tab
+
+        Args:
+            tab: The QWebView to be closed.
+
+        Return:
+            True if tab can be destroyed and false otherwise.
+        """
```

Comment:
Two minor nitpicks:
* Add a period after the first line
* Replace `false` by `False`.